zebra-test/Cargo.toml

```
@@ -2,7 +2,9 @@
 name = "zebra-test"
 version = "1.0.0-beta.25"
 authors = ["Zcash Foundation <zebra@zfnd.org>"]
+description = "Miscellaneous test code for Zebra"
```

Nitpick: "Miscellaneous" and "utilities" don't actually mean much

```suggestion
description = "Test harnesses and test vectors for Zebra"
```